pkg/services/ngalert/models/admin_configuration_test.go

```diff
@@ -2,62 +2,11 @@ package models
 
 import (
 	"errors"
-	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 )
 
-func TestAdminConfiguration_AsSHA256(t *testing.T) {
```

Comment:
It seems we moved some of the redaction code to `router.go`, but only deleted the tests. Even though it's only used for logs now, could we still add a corresponding test on `buildRedactedAMs` in `router_test.go`? That way we at least have some extra protection around accidentally logging sensitive info in URLs there.

Reply:
Of course